shared/trainer/defaults.yaml

```diff
@@ -93,7 +94,7 @@ gen:
   m: # specific params for the mask-generation decoder
     <<: *default-gen
     output_dim: 1
-    use_advent: True
+    use_advent: False
```

Comment:
is there a reason for this change of default?

Reply:
Without advent seems to work better than with advent at the moment. The defaults should reflect our best performance right?